commands/http/handler.go

```diff
@@ -56,10 +57,8 @@ const (
 )
 
 var localhostOrigins = []string{
-	"http://127.0.0.1",
-	"https://127.0.0.1",
-	"http://localhost",
-	"https://localhost",
+	"127.0.0.1:5001",
+	"localhost:5001",
 }
```

Comment:
:+1: to @mappum's comment. should try to grab it from config.

i think this is the key part: adding the API port, not removing the protocol. this should work then:

```go
var localhostOrigins = []string{
 "http://127.0.0.1:<api-port>", 
 "https://127.0.0.1:<api-port>",
 "http://localhost:<api-port>",   
 "https://localhost:<api-port>",    
 }
```